shared/ui-components/src/footers/SiteFooter/SiteFooter.stories.tsx

```diff
@@ -49,9 +52,6 @@ export const withLogoAndCredits = () => (
   <SiteFooter>
     <FooterSection>{logo}</FooterSection>
     <FooterSection>{credits}</FooterSection>
-    <FooterSection>
-      <img className="h-16 w-16" src="/images/eho-logo-white.svg" alt="Equal Housing Opportunity Logo" />
-     </FooterSection>
     <FooterNav copyright="Alameda County © 2020 • All Rights Reserved">
```

Comment:
I somehow never noticed that we had Alameda hard coded into the story here (and into the reference implementation FooterNav). Can we take this opportunity to clean it up so the reference implementation and ui-components are generic?